ppml/trusted-big-data-ml/scala/docker-occlum/run_spark_on_occlum_glibc.sh

```diff
@@ -102,13 +102,24 @@ init_instance() {
            echo "[ERROR] Attestation set to true but NO PCCS"
            exit 1
         else
-           export ENABLE_SGX_DEBUG=false
-           sed -i "s#https://localhost:8081/sgx/certification/v3/#${PCCS_URL}#g" /etc/sgx_default_qcnl.conf
+           echo 'PCCS_URL='${PCCS_URL}'/sgx/certification/v3/' > /etc/sgx_default_qcnl.conf
```

Comment:
Only do this when ATTESTATION=true. https://github.com/intel-analytics/BigDL/blob/main/ppml/trusted-big-data-ml/python/docker-graphene/entrypoint.sh#L79

Reply:
it just works as you say.